### PR TITLE
upload to s3

### PR DESCRIPTION
**Pull request by DaytonG**

### CODE REVIEW

**Comment by DaytonG** [comment]
the file its uploading is junk, but that's an easy change when there's something to upload

File: azure-pipelines.yml [hunk 1]
```diff
@@ -10,24 +10,85 @@ pool:
...
+        vmImage: "ubuntu-18.04"
+      steps:
+        # do not checkout code. we only need to upload the published artifact
+        - checkout: none
+        - download: current
+          artifact: installer
+        - task: Bash@3
+          env:
+            AWS_ACCESS_KEY_ID: $(accessKey)
+            AWS_SECRET_ACCESS_KEY: $(secretKey)
```
Comment (jvilhuber) [inline azure-pipelines.yml:62]: Nice. Gets passed in, eh?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -10,24 +10,85 @@ pool:
...
+                exit 0
+              fi
 
-- task: VSTest@2
-  inputs:
-    platform: '$(buildPlatform)'
-    configuration: '$(buildConfiguration)'
+              set -x
+              
+              echo "install AWS CLI"
```
Comment (jvilhuber) [inline azure-pipelines.yml:90]: I guess there's no native push to s3 in azure, eh? Yea, this works, I guess.

File: azure-pipelines.yml (hunk 1 above)
Comment (DaytonG) [reply azure-pipelines.yml:62]: Yea, it's part of the variable group I import in the "variables" section. Unfortunately that has to be created manually... but it's a one-time thing... so whatever.

**Comment by DaytonG** [comment]
Will merge this in when we have something real to push

**Comment by DaytonG** [comment]
doing this work in another branch

**Pull request closed without merging**